cmd/kubeadm/app/master/apiclient.go

```diff
@@ -103,6 +104,15 @@ func CreateClientAndWaitForAPI(adminConfig *clientcmdapi.Config) (*clientset.Cli
 	return client, nil
 }
 
+func isNodeReady(node *api.Node) bool {
```

Comment:
I find it strange that there is no such utility function in the `api/v1` api client, is there a reason for this? I can't use the one from the internal `api` as the `Node` in the signature differs.